index.js

```diff
@@ -142,7 +141,7 @@ File.prototype.inspect = function() {
 };
 
 File.isVinyl = function(file) {
-  return (file && file._isVinyl === true) || false;
+  return Boolean(file && file._isVinyl);
```

Comment:
In old versions of vinyl, that property doesn't exist. We should be casting with `!!` though.